testdata/access_token_response.json

```diff
@@ -0,0 +1 @@
+{"data":{"response":{"token":"hidden"}}}
```

Comment:
Wouldn't it be easier to just pass it as a string?

Reply:
I just wanted to be consistent and store all the mocked response files in the `testdata` folder.
But if you think it's better to use a string here - that's also fine.